.github/workflows/sync.yml

```diff
@@ -0,0 +1,52 @@
+name: Sync
+
+on:
+  push:
+    branches: [master, staging]
+  workflow_dispatch:
+
+jobs:
+  sync-master:
+    runs-on: ubuntu-latest
+    if: github.ref == 'refs/heads/master'
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 15
+      - name: Install awscli
+        run: pip install --user awscli
+      - name: 📦 Install Dependencies
+        run: yarn install --immutable --immutable-cache
+      - name: 🔨 Build
+        run: yarn build
+      - name: 🚀 Sync to S3
+        run: |
+          ~/.local/bin/aws s3 sync dist s3://cru-ert-web-prod --region us-east-1 --acl public-read --exclude '*.html' --cache-control 'public, max-age=31536000',
+          ~/.local/bin/aws s3 sync dist s3://cru-ert-web-prod --region us-east-1 --acl public-read --include '*.html' --cache-control 'public, max-age=180'
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
```

Comment:
Looks like you don't have these setup at https://github.com/CruGlobal/conf-registration-web/settings/secrets/actions yet. Might be a good excuse to rotate the keys since Travis won't need them after this is merged.

The #aws Slack channel has an email in the channel description you can use for getting help creating keys (thought dumping the email in a public repo might be weird). They can delete the old keys after we confirm the Github Actions deploy is working.

Reply:
Yeah, I was thinking it would be good to wait to get them until this pr is good to go. But thanks for letting me know how to go about getting new ones because I was unsure 🙂